### ISSUE TITLE
F10 Doesn't Work

### ISSUE THREAD

**Issue by nbarkhina** [MicrosoftDocs/vscodespaces#820]
When doing debugging, pressing the F10 keyboard key doesn't Step Over as it should but instead puts focus on the Application Icon as in the screenshot below. I am using Github Codespaces in the Chrome Browser on Windows 10.

![image](https://user-images.githubusercontent.com/10982154/95654588-eebf7e80-0ace-11eb-8b3a-b43ddbec8e6d.png)

**Comment by lostintangent** [comment]
@bpasero Thoughts? I was able to repro the behavior that @nbarkhina mentioned?

**Comment by bpasero** [comment]
@sbatten did we recently add `F10` to focus the menu as an accessibility feature maybe? This could also be a reserved keybinding of the browser after all similar to `Cmd+W` that we cannot take.

**Comment by sbatten** [comment]
@bpasero yes, discussion here https://github.com/microsoft/vscode/issues/82759
